Cargo.toml

```diff
@@ -73,6 +73,11 @@ opt-level = 3
 opt-level = 3
 lto = "thin"
 
+# ugly hack since doctests link against debug version of rustpython-vm even
+# when profile.test is optimized
```

Comment:
Though we usually use optimization and debug option exclusively, optimization is about optimization itself and debug is about debug information. So I think giving optimization option to test doesn't change it is a debug or not.

Reply:
Right, and it does work fine unoptimized on our machines apparently, but I think there's some timeout in ci that kills the test process if it takes too long? And afaict it was linking against release in Rust 1.56, the update broke it a bit, this just brings it back to previous behavior

Reply:
ach wait the tests aren't even fixed, dang it

Reply:
yep, I think it doesn't fix anything unless it is related to cache

Reply:
ugh I don't know why I didn't consider that it was a cache issue

Reply:
The cache action is reporting a cache *miss*, so I doubt that it's actually a cache issue.

Reply:
Could https://github.com/rust-lang/rust/issues/92869 be related?

Reply:
Ahh, probably